default.env

```diff
@@ -0,0 +1,5 @@
+# Rename to `.env` to use.
+# These will not be used if an environment variable by the same name exists.
+# Only variables named `JORE_UI_*` or ones defined in `env.js` are exported to frontend application by default.
+PORT = 3000 # Port to serve ui from
+API_URL = http://localhost:3040 # URL ui should use for backend api
```

Comment:
Comments should be on their own lines. Doesn't work for me if it's on the samel line with the actual variable